src/cmd_argument.rs

```diff
@@ -121,3 +122,33 @@ impl CmdArgument for LogCommand {
         config.log_command = true;
     }
 }
+
+/// Please, see the [`CmdArgument`] implementation for [`CurrentDir`] below.
```

Comment:
I think you can leave out the "Please," it sounds a little desperate 😜

Reply:
https://github.com/soenkehahn/stir/pull/40/commits/007553105ef574bce559446631d7686aebc7b86d